pom.xml

```
@@ -485,6 +490,27 @@
                     <artifactId>protoc-jar-maven-plugin</artifactId>
                     <version>${version.protoc.maven.plugin}</version>
                 </plugin>
+                <plugin>
+                    <groupId>net.revelc.code.formatter</groupId>
+                    <artifactId>formatter-maven-plugin</artifactId>
+                    <version>${version.code.formatter}</version>
+                    <configuration>
+                        <configFile>${maven.multiModuleProjectDirectory}/support/ide-configs/eclipse/debezium-formatter.xml</configFile>
+                        <skip>${format.skip}</skip>
+                    </configuration>
+                </plugin>
+                <plugin>
+                    <groupId>net.revelc.code</groupId>
+                    <artifactId>impsort-maven-plugin</artifactId>
+                    <version>${version.impsort}</version>
+                    <configuration>
+                        <groups>java.,javax.,org.,com.,io.</groups>
+                        <staticGroups>*</staticGroups>
+                        <staticAfter>true</staticAfter>
```

```suggestion
                        <staticAfter>false</staticAfter>
```

I played a bit with this setting and it turns out that currently more files have static import before non-static ones (72 changed files vs. 162 changed files). So let's keep statics first.

---

So I pushed a change for this commit and also applying the import re-org actually.